ramls/order.raml

```diff
@@ -204,3 +204,19 @@ resourceTypes:
     is: [validate]
     post:
       description: Create piece record
+    /{id}:
+      uriParameters:
+        id:
+          description: The UUID of a piece record
+          type: UUID
+      is: [validate, language]
+      put:
```

Comment:
Please declare all possible response types for both `put` and `delete`

Reply:
Done

Reply:
`404` is also one of the responses for both operations.
Also `application/json` content type will be used if an error handled by mod-orders logic.